packages/ndla-editor/src/SlateBlockMenu.js

```diff
@@ -154,7 +154,7 @@ const SlateBlockMenu = React.forwardRef(
         ref={ref}
         css={[buttonCSS, isOpen && buttonOpen]}
         data-cy={cy}
-        onClick={() => onToggleOpen(!isOpen)}>
+        onMouseDown={() => onToggleOpen(!isOpen)}>
```

Comment:
Vet vi hvorfor dette fikser buggen? Føles litt som et plaster å fikse en bug med å endre oppførsel. 
Jeg er helt med på å endre oppførselen om det er det vi vil da :)

Reply:
Det var ikke helt "bug" til å begynne med, var bare det at når man trykka for treig på knappen så forsvant den fordi at animasjonen var laget sånn i koden, for å slippe å fikse mye av hele update pipeline og andre steder så er det en quick fix til det vi ønsket, altså at man får menyen alltid, selvom man er treig å trykke fordi nå kommer menyen til å poppe opp instant å ikke etter at man releaser museknappen.

Reply:
Jeg skjønner.
Hvorfor starter animasjonen på press og ikke release?

Reply:
Fordi at den triggres fra componentDidUpdate() funksjonen, altså for hver gang noe blir trykka på, og da når knappen blir trykka på starter den animasjonen altså på trykk ikke release.

Reply:
Vi kan fortsette der [LINK](https://github.com/NDLANO/editorial-frontend/pull/652)